src/bootstrap.php

```diff
@@ -284,4 +290,8 @@
     }
 });
 
+$app->error(function (Throwable $e, Request $request, int $code) use ($app) {
+    $app['logger']->error("$code response on {$request->getUri()}", ['exception' => $e]);
```

Comment:
Probably don't want to `error` 4xx responses?

Check if `$e` is a `ApiProblemException` or `HttpExceptionInterface` and grab the code (don't think `$code` is (always) the HTTP status code).